x/liquidity/spec/06_end_block.md

```diff
@@ -1,23 +1,28 @@
-<!--
-order: 6
--->
+<!-- order: 6 -->
 
-# Before-End-Block
+ # Before-End-Block
 
-## 1) Append messages to LiquidityPoolBatch
+## Append messages to LiquidityPoolBatch
 
-After successful message verification and coin `escrow` process, the incoming `MsgDepositWithinBatch`, `MsgWithdrawWithinBatch`, and `MsgSwapWithinBatch` are appended into the current `PoolBatch` of the corresponding `Pool`
+After successful message verification and coin `escrow` process, the incoming `MsgDepositWithinBatch`, `MsgWithdrawWithinBatch`, and `MsgSwapWithinBatch` messages are appended to the current `PoolBatch` of the corresponding `Pool`.
 
 # End-Block
 
-## 1) Execute LiquidityPoolBatch upon its execution heights
+## Execute LiquidityPoolBatch upon execution heights
 
-If there are `{*action}MsgState`s that is not yet executed in the `PoolBatch` for each `Pool`, the `PoolBatch` is executed. It could contains `DepositLiquidityPool`, `WithdrawLiquidityPool` and `SwapExecution` process.
+If there are `{*action}MsgState` messages that have not yet executed in the`PoolBatch` for each `Pool`, the`PoolBatch`is executed. This batch could contain `DepositLiquidityPool`, `WithdrawLiquidityPool`, and `SwapExecution` process.
 
-## 1-A) Transact and Refund for each message
+### Transact and refund for each message
 
-Transactions are made through `escrow`, and refunds are made for cancellations, partial cancellations, expiration, and failed messages.
+Transactions are made through `escrow`. Refunds are made for cancellations, partial cancellations, expiration, and failed messages.
 
-## 1-B) Set states for each message according to the results
+### Set states for each message according to the results
 
-After `1-A`, Update the state of each `{*action}MsgState` according to the results. Even If the message is completed or expired, Set the `ToBeDeleted` state value as true instead of deleting the message directly from the `end-block` and then delete the messages which have `ToBeDeleted` state from the begin-block in the next block, so that each message with result state in the block can be stored to kvstore, for the past messages with result state can be searched when kvstore is not pruning.
+After transact and refund for each message, update the state of each `{*action}MsgState` message according to the results.
+
```

Comment:
another reason not to number and to use headings is when we reference earlier content, using a number adds another complexity... using heading and context is almost always the right structure for content. 

I added numbered list items for sequential actions below

Reply:
sounds good!